### ISSUE TITLE
problem of the prediction

### ISSUE THREAD

**Issue by liuxingbin** [ChaoningZhang/MobileSAM#109]
When I generated the full mask following https://github.com/ChaoningZhang/MobileSAM#demo, I found that using `model.eval()` differs from the results of not using `model.eval()`. Interestingly, model.eval() is already settled at https://github.com/ChaoningZhang/MobileSAM/blob/master/mobile_sam/build_sam.py#L88, and I checked `model.training` is False.
What's the problem?
